.github/actions-labeler.yml

```diff
@@ -15,6 +15,7 @@
 
 "area: dds: tree":
   - experimental/dds/tree/**
+  - experimental/dds/tree2/**
```

Comment:
we could probably remove the labeler from the old path right now if we want, but I figured I'd leave it to be safe--I'm not sure how "source of truth" works for which labelling config is used for a given branch. If it's like pipelines we could remove.